tests/Integration/FontTest.php

```diff
@@ -340,14 +340,39 @@ public function testDecodeText()
             ],
         ];
         $this->assertEquals('æöü', $font->decodeText($commands));
+    }
 
-        $commands = [
-            [
-                't' => '<',
-                'c' => 'C3A6C3B6C3BC', //Unicode encoded string
-            ],
-        ];
-        $this->assertEquals('æöü', $font->decodeText($commands));
+    /**
+     * Font could have indirect encoding without `/Type /Encoding`
+     * which would be instance of PDFObject class (but not Encoding or ElementString).
+     *
+     * @see https://github.com/smalot/pdfparser/pull/500
+     *
+     * @return void
+     */
+    public function testDecodeTextForFontWithIndirectEncodingWithoutTypeEncoding()
```

Comment:
```suggestion
     */
    public function testDecodeTextForFontWithIndirectEncodingWithoutTypeEncoding(): void
```

Reply:
By the way, I added a return type for all test's methods.

Reply:
Thats good